transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -272,9 +263,7 @@ public ChannelFuture bind(InetAddress inetHost, int inetPort) {
      */
     public ChannelFuture bind(SocketAddress localAddress) {
         validate();
-        if (localAddress == null) {
-            throw new NullPointerException("localAddress");
-        }
+        ObjectUtil.checkNotNull(localAddress, "localAddress");
         return doBind(localAddress);
```

Comment:
@qeesung: nit you can merge the above 2 lines

Reply:
Done